src/accel/LocalTransporter.hh

```diff
@@ -102,7 +102,8 @@ class LocalTransporter
     UniqueEventId event_id_;
 
     size_type auto_flush_{};
-    size_type max_steps_{};
+    size_type max_step_iters_{};
+    double buffer_energy_{0};
```

Comment:
Was this `real_type` or `double`? Still makes sense, store it as double in the cpu thread and cast to `real_type` on device, but wanted to make sure.

Reply:
It's never used on the device, just for a message, so it probably doesn't matter...